README.md

```diff
@@ -29,7 +32,20 @@ Instantiate an instructor named 'Robin' and call its `givesHomework` method with
 Your Answer:
 
 ```js
-// your code here
+
+var Instructor = function(name, assignment) {
+  this.name = name;
+  this.assignment = assignment;
+});
+
+Instructor.prototype.givesHomework = function() {
+  console.log(this.name + " gives the students " + this.assignment + " for Friday's homework.");
+};
+
+var robin = new Instructor("Robin", "Intro to Ruby");
+
+robin.givesHomework;
+
 
```

Comment:
In this scenario, assignment does not need to be passed in as an argument to the constructor. Instead, it should be passed in as an argument to the `givesHomework` method.